mne_bids/tests/test_write.py

```diff
@@ -3151,13 +3151,16 @@ def test_anonymize(subject, dir_name, fname, reader, tmp_path, _bids_validate):
     # handle different edge cases
     if subject == "emptyroom":
         bids_path.update(task="noise", session=raw_date, suffix="meg", datatype="meg")
+        erm = dict()
     else:
         bids_path.update(task="task", suffix="eeg", datatype="eeg")
+        # make sure anonymization works when also writing empty room file
+        erm = dict(empty_room=raw.copy())
     daysback_min, daysback_max = get_anonymization_daysback(raw)
     anonymize = dict(daysback=daysback_min + 1)
     orig_bids_path = bids_path.copy()
     bids_path = write_raw_bids(
-        raw, bids_path, overwrite=True, anonymize=anonymize, verbose=False
+        raw, bids_path, overwrite=True, anonymize=anonymize, verbose=False, **erm
```

Comment:
I think this can be simplified to

```python
if subject == "emptyroom":
    ...
    er_raw=None
else:
    ...
    er_raw = raw.copy()
...
write_raw_bids(..., empty_room=er_raw)
```
or so

Reply:
Sure, that's a slight improvement I guess

Reply:
I think we can merge this as is though, I think the difference in code here is miniscule. Thanks for the PR and the reviews.

Reply:
i don't know, why would you create a dict if you don't need it? just to later unpack the only element it contains 🧐

Reply:
@hoechenberger see #1274